octopus/src/main/scala/octopus/ValidationResult.scala

```diff
@@ -44,7 +44,7 @@ sealed class ValidationResult[T](private[octopus] val value: T, val errors: List
     }
 
   def toTaggedEither[Tag]: Either[List[ValidationError], T @@ Tag] =
-    toEither.right.map(tag[Tag](_))
```

Comment:
Note we don't have `.map` directly on Either in Scala 2.11. Thus we need to go back to using `.right` projection which seems to work also in Scala 2.13 (although it may require turning off `-Xfatal-warnings` which is fine for me).

Reply:
I preserved `-Xfatal-warnings` and I don't use `.right`. See the line below. 🙂

Reply:
Even better 👍